tests/integration/test_data_store_mgr.py

```diff
@@ -13,22 +13,103 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from copy import copy, deepcopy
 import pytest
 
+from cylc.flow import ID_DELIM
 from cylc.flow.data_store_mgr import (
     FAMILY_PROXIES,
+    JOBS,
     TASKS,
     TASK_PROXIES,
-    WORKFLOW
+    WORKFLOW,
+    JOB_STATUSES_ALL
+)
+from cylc.flow.task_state import (
+    TASK_STATUS_FAILED,
+    TASK_STATUS_SUCCEEDED,
+    TASK_STATUS_WAITING
 )
+from cylc.flow.wallclock import get_current_time_string
+
+
+def job_config(schd):
+    return {
+        'owner': schd.owner,
+        'host': 'commet',
+        'submit_num': 3,
+        'task_id': 'foo.1',
+        'job_runner_name': 'background',
+        'env-script': None,
+        'err-script': None,
+        'exit-script': None,
+        'execution_time_limit': None,
+        'init-script': None,
+        'post-script': None,
+        'pre-script': None,
+        'script': 'sleep 5; echo "I come in peace"',
+        'work_d': None,
+        'directives': {},
+        'environment': {},
+        'param_var': {},
+        'logfiles': [],
+        'platform': {'name': 'platform'},
+    }
+
+
+@pytest.fixture
+def job_db_row():
+    return [
+        '1',
+        'foo',
+        'running',
+        4,
+        '2020-04-03T13:40:18+13:00',
+        '2020-04-03T13:40:20+13:00',
+        '2020-04-03T13:40:30+13:00',
+        'background',
+        '20542',
+        'localhost',
+    ]
+
+
+@pytest.fixture
+@pytest.mark.asyncio
+async def myflow(flow, scheduler):
+    schd = scheduler(reg)
+    await schd.install()
+    await schd.initialise()
+    await schd.configure()
+    return schd
```

Comment:
Unused?

Reply:
Yes. `update_workflow` is called within `update_data_structure` and `initiate_data_model`, so well covered in integration...